cnnimg.py

```diff
@@ -31,10 +31,7 @@
 import Image as img
 import numpy as np
 import os.path
-import warnings
 
-#Ignore warnings
```

Comment:
Numpy and Scipy throws several intermittent warnings without this statement. It's better to display warnings only in the debug mode. A debug option (by default, false) can be added in the library to display warnings selectively. The debug option can be set after importing the library. Do you want make this change?